code/modules/spacepods/construction.dm

```diff
@@ -1,4 +1,4 @@
-/obj/spacepod/examine(mob/user)
+/obj/spacepod/examine(mob/user) //these output above the spacepod's text block for some reason. Not good.
```

Comment:
A. Spacepods 😔 
B. Is this with your spacepod specifically or in general. It seems like the latter.

Reply:
If so, I'd recommend removing this as a change and making a bug report and request the "reproduceable" tag.

Reply:
I must've missed that when I was unpodding the ship. Gonna remove this.